frontend/packages/kubevirt-plugin/src/components/ssh-service/SSHForm/SSHFormKey/ssh-form-key.scss

```diff
@@ -0,0 +1,33 @@
+.SSHFormKey-show-text-area {
```

Comment:
@matthewcarleton can you review the CSS?
we should not use these exact number values.

Can't you use a PF componenst instead @metalice ?